blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -195,6 +201,7 @@ const ExtraLargePromo = ({ customFields }) => {
                               mediumHeight={ratios.mediumHeight}
                               largeWidth={ratios.largeWidth}
                               largeHeight={ratios.largeHeight}
+                              client={imageConfig === 'resize-image-api-client'}
```

Comment:
what's happening here? new prop for image component. may have missed this in instructions

Reply:
This is needed for lazy load, for blocks that are using lazy load custom field and also use the fallback placeholder block the fallback image runs through the image resizer content source. So with out this the fallback image would fail

Reply:
is client added to the <Image component?

Reply:
oh, ok. this is being added to the PlaceholderImage block not the Image block.